patterns/disposer.md

```diff
@@ -0,0 +1,84 @@
+# The Promise Disposer Pattern
```

Comment:
> I would say its an established pattern in other languages

we are talking specifically about promise-based patterns. `finally` is already a part of javascript